bin/pullRawDataFromDS.sh

```diff
@@ -257,8 +257,31 @@ else
 	# Cleanup old data if data transfer with rsync finished successfully (and hence did not crash this script).
 	#
 	log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "Deleting data older than 14 days from ${HOSTNAME_DATA_STAGING%%.*}:/groups/${GROUP}/${SCR_LFS}/ ..."
-	# shellcheck disable=SC2029
-	/usr/bin/ssh "${HOSTNAME_DATA_STAGING}" "/bin/find /groups/${GROUP}/${SCR_LFS}/ -mtime +14 -ignore_readdir_race -delete"
+	## get the project name by parsing the /home/umcg-ndewater/files/ folder, directories only and no empty or '.'
+	readarray -t gsProjectsSourceServer< <(rsync -f"+ */" -f"- *" "${HOSTNAME_DATA_STAGING}:/home/umcg-ndewater/files/" | awk '{if ($5 != "" && $5 != "."){print $5}}')
+	if [[ "${#gsProjectsSourceServer[@]:-0}" -eq '0' ]]
+	then
+		log4Bash 'WARN' "${LINENO}" "${FUNCNAME:-main}" '0' "No sample sheets found at ${HOSTNAME_DATA_STAGING}:/home/umcg-ndewater/files/"
+	else
+		for gsProject in "${gsProjectsSourceServer[@]}"
+		do
+			gsProject="$(basename "${gsProject}")"
+			
+			# convert date to seconds to have an easier calculation of the date difference			
+			dateInSecProject="$(date -d"$(rsync "${HOSTNAME_DATA_STAGING}:/home/umcg-ndewater/files/${gsProject}" | awk '{print $3}')" +%s)"
+			dateInSecNow=$(date +%s)
+			#86400 = 1 day in seconds 
```

Comment:
Zonder spatie na # verwacht ik een commando dat uigecomment is, maar dit is een omschrijving/zin en geen commando.